src/services/activityReports.js

```diff
@@ -311,12 +310,10 @@ export function activityReports(
 ) {
   const scopes = filtersToScopes(filters);
 
-  const endDte = moment().format('MM/DD/yyyy');
-
   const where = {
     calculatedStatus: REPORT_STATUSES.APPROVED,
     [Op.and]: scopes,
-    startDate: { [Op.gte]: '2020-08-31', [Op.lte]: endDte },
+    startDate: { [Op.gte]: '2020-09-01' },
```

Comment:
I think this change was added as part of TTAHUB-321 see [files changed](https://github.com/adhocteam/Head-Start-TTADP/pull/454/files).

Reply:
looks like in those files changed you linked it was ```[Op.gte]: 2020-09-01``` as well, so it must have been tweaked to ```2020-08-31``` in the meantime. 

we don't need to specify not showing start dates in the future, right? since we are only fetching approved reports?